recipes/_configure_server.rb

```diff
@@ -178,7 +179,7 @@
   server_stanzas['shclustering']['pass4SymmKey'] = CernerSplunk::ConfTemplate.compose encrypt_password, CernerSplunk::ConfTemplate::Value.constant(value: pass) if pass
   server_stanzas['shclustering']['conf_deploy_fetch_url'] = deployer_uri
   server_stanzas['shclustering']['disabled'] = 0
-  server_stanzas['shclustering']['mgmt_uri'] = "https://#{node['splunk']['mgmt_host']}:8089"
+  server_stanzas['shclustering']['mgmt_uri'] = "https://#{management_ipaddress}:8089"
```

Comment:
I just realized this is a non-passive change that I believe would require a major version bump of the cookbook. It will break existing search head clusters without additional configuration changes on the consumers end. What was the reason behind this change from the configurable management host to pulling the ip address from the ohai data?

Reply:
Not sure that is correct, `node['splunk']['mgmt_host']` = `node[ipaddress]` which I believe was the ipaddress of the default_interface.

By default `management_ipaddress` is the same thing, but this allowed us to not have to use the vagrant ohai plugin. 

In the future would allow us specify an interface to use for something specific. So the boxes with many interfaces each interface could be used for something specific rather than one interface doing all of the work.

At least I think that was all of the reasons behind it, @acharlieh could correct me if I am wrong.

Reply:
Understood, so which path should I go down?
Removed the recent work I did to move towards making testing work for both chef 12 and 14 or find a way to make it not a breaking change?